src/status_im/contexts/wallet/swap/events.cljs

```diff
@@ -152,11 +156,10 @@
                                   #(cond-> %
                                      :always
                                      (assoc
-                                      :last-request-uuid      request-uuid
-                                      :amount                 amount
-                                      :amount-hex             amount-in-hex
-                                      :loading-swap-proposal? true
-                                      :initial-response?      true)
+                                      :last-request-uuid request-uuid
+                                      :amount            amount
+                                      :amount-hex        amount-in-hex
+                                      :initial-response? true)
```

Comment:
Just removing `:loading-swap-proposal? true` from the assoc, loading state is now set by a new event `:wallet.swap/set-loading-swap-proposal`.